### ISSUE TITLE
Add websocket session identifier to listen verb websocket headers

### ISSUE THREAD

**Issue by alex-wirebee** [jambonz/jambonz-feature-server#965]
Use case: Have multiple replicas of a websocket server listening for bidirectional streams, connected to from the jambonz listen verb. 
A load balancer is used in front of these replicas, and in order for the load balancer to always forward a websocket session to a specific replica ([header based session affinity](https://developers.cloudflare.com/load-balancing/understand-basics/session-affinity/#header)), the websocket headers need to have an id (in this case, call_sid would probably be a good identifier) which the load balancer can map to a replica.

**Comment by davehorton** [comment]
are you specifically asking for this to be carried in an HTTP custom header ?

**Comment by alex-wirebee** [comment]
Yes, I believe that should do it since the load balancer lets me specify which header to use for identifying the session

**Comment by davehorton** [comment]
why do you care which replica a particular call session websocket connection gets routed to?
